django_celery_beat/models.py

```diff
@@ -27,21 +27,21 @@
 SECONDS = 'seconds'
 MICROSECONDS = 'microseconds'
 
-PERIOD_CHOICES = (
+PERIOD_CHOICES = [
     (DAYS, _('Days')),
     (HOURS, _('Hours')),
     (MINUTES, _('Minutes')),
     (SECONDS, _('Seconds')),
     (MICROSECONDS, _('Microseconds')),
-)
+]
 
-SINGULAR_PERIODS = (
+SINGULAR_PERIODS = [
     (DAYS, _('Day')),
     (HOURS, _('Hour')),
     (MINUTES, _('Minute')),
     (SECONDS, _('Second')),
     (MICROSECONDS, _('Microsecond')),
-)
+]
```

Comment:
Nice work!!

Can we please change this only on PyPy and in a way that allows it to be removed after PyPy v7.3.14 is released?

Tuples take less memory and signal to the reader that these values should normally not be modified at runtime.  Also, given comments in the first few episodes of the [core.py podcast](https://podcasters.spotify.com/pod/show/corepy), tuples should perform better in the multi-GIL world of Python 3.13+.

Something like...
```
import platform

if platform.python_implementation() == "PyPy":  # Workaround for PyPy < v7.3.14
    PERIOD_CHOICES = list(PERIOD_CHOICES)
    SINGULAR_PERIODS = list(SINGULAR_PERIODS)
```

Reply:
Then why is there a bug at all?

Reply:
It's all about `test_models_match_migrations` that was failing in your PR https://github.com/celery/django-celery-beat/actions/runs/7267638477/job/19801878411?pr=705#step:6:2073

I think the PR description should cover the problem pretty well. If it's unclear, let me know and I'll fix it.